params/variants.toml

```diff
@@ -5355,25 +5355,60 @@ sampler = "ф"
 samplerExplain = "Cyrillic Lower Ef"
 tagKind = "letter"
 
-[prime.cyrl-ef.variants.serifless]
+[prime.cyrl-ef.variants-buildup]
+entry = "bowl"
+descriptionLeader = "Cyrillic Lower Ef (`ф`)"
+
+[prime.cyrl-ef.variants-buildup.stages.bowl."*"]
+next = "bar"
+
+[prime.cyrl-ef.variants-buildup.stages.bowl.standard]
 rank = 1
-description = "Cyrillic Lower Ef (`ф`) with standard shape and without serifs"
-selector."cyrl/ef"  = "serifless"
+keyAffix = ""
+selectorAffix."cyrl/ef" = ""
+selectorAffix."cyrl/ef.BGR" = ""
 
-[prime.cyrl-ef.variants.cursive]
+[prime.cyrl-ef.variants-buildup.stages.bowl.split]
 rank = 2
-description = "Cyrillic Lower Ef (`ф`) with cursive shape"
-selector."cyrl/ef"  = "cursive"
+descriptionAffix = "split bowl"
```

Comment:
You are making breaking changes.
Add `nonBreakingVariantAdditionPriority = 100` to prevent that.